README.md

```diff
@@ -65,11 +65,81 @@ This section will show how to use the opinionated framework `ex_gram` for telegr
 
 ### Creating a bot!
 
-WIP
+Creating a bot is pretty simple, you can use the `mix bot.new` task to setup your bot. For example:
+
+``` shell
+$ mix new my_bot --sup
+$ cd my_bot
+```
+
+Put `ExGram` and a JSON engine (`Jason` for example) as dependencies in your project like so:
+
+``` elixir
+# Run "mix help deps" to learn about dependencies.
+defp deps do
+  [
+    {:ex_gram, "~> 0.7"},
+    {:jason, "~> 1.0.0"}
+  ]
+end
+```
+
+Get the project deps and run the bot new task:
```

Comment:
I would not specify here how to add it to the project, I would point to the #installation section